source/documentation/reference/getting-help.html.md.erb

```diff
@@ -12,6 +12,19 @@ review_in: 3 months
 The quickest way to get help is to post a question in the
 [#ask-cloud-platform] Slack channel.
 
+## Hammer-bot
+
+The hammer-bot is a Slack bot that reports pull request check state to raised pull requests in the ask-ckoud-platform channel. 
+It will report the state with a reaction in slack on the comment
+
+- ✨ passed
+- ❌ failed
+- 🔁 pending
+- ⏳️ running for more 10 minutes
+
+if you check has been running for more than 10 minutes please go to your pull request to check why, if its concourse this is likely 
+due to concourse not picking up the pull request, you can trigger the check by pushing a blank commit to the pull request.
```

Comment:
Not sure this should be on a seperate line.

Reply:
this is ok its a md format so it takes i as one line